benchmark/fs/bench-opendirSync.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const bench = common.createBenchmark(main, {
+  type: ['existing', 'non-existing'],
+  bufferSize: [ 4, 32, 128 ],
+  n: [1e4],
+});
+
+function main({ n, type, bufferSize }) {
+  let path;
+
+  switch (type) {
+    case 'existing':
+      path = 'lib';
```

Comment:
Can we use different paths for different iterations? For example creating them before setting up the benchmark, or just iterate through the test directory (the latter might give you a less-micro benchmark that can be closer to how people use it in the real world).

Reply:
Agreed. I have to reduce the iteration to `1e3`. It takes 5+ minutes on M1 Max with 64GB mem.